test/game_test.rb

```diff
@@ -38,8 +38,7 @@ def test_it_has_attributes
     assert_equal   '/api/v1/venues/null' , @game.venue_link
   end
 
-  def def_test_it_can_read_games_from_CSV
```

Comment:
Caused this not to ever run because it's def def_test_(stuff), not def test_(stuff)